.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: "Build"
+on:
+    push:
+      branches:
+        - main
+    pull_request:
+
+jobs:
+  build:
+    name: Build Job ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        java: ["11"]
```

Comment:
You cannot compile an iOS sample using `ubuntu`, maybe this should run only on `macos`?